resources/views/layouts/base.blade.php

```diff
@@ -21,26 +21,29 @@
     @include('layouts._favicons')
     @include('layouts._cookie_consent')
     @include('layouts._google_analytics')
+    @include('layouts._ads._ad_sense')
+    <script src="{{ mix('js/app.js') }}" defer></script>
```

Comment:
Maybe put the deferred script in the same location as the UI stub: https://github.com/laravel/ui/blob/1.x/src/Auth/bootstrap-stubs/layouts/app.stub#L13